help/destinations/catalog/advertising/snap-inc.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Snap Inc
+description: Use your audience segments in the Snapchat Ads Platform.
```

Comment:
Minor rewording suggestion: Learn how to connect to the Snapchat Ads Platform and export your audience segments from Experience Platform